src/app/features/auth/components/forgot-password/forgot-password.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnDestroy, Signal, WritableSignal, computed, effect, inject, signal } from '@angular/core';
+import { FormControl, FormGroup, FormsModule, NonNullableFormBuilder, ReactiveFormsModule, Validators } from '@angular/forms';
+
+import { NzFormModule } from 'ng-zorro-antd/form';
+import { NzInputModule } from 'ng-zorro-antd/input';
+import { NzButtonModule } from 'ng-zorro-antd/button';
+import { NzGridModule } from 'ng-zorro-antd/grid';
+import { NzTypographyModule } from 'ng-zorro-antd/typography';
+import { NzSpaceModule } from 'ng-zorro-antd/space';
+import { Router, RouterModule } from '@angular/router';
+import { AuthService } from '../../../../shared/services/auth.service';
+import { EMPTY, Subject, catchError, finalize, takeUntil, tap } from 'rxjs';
+import { NzMessageModule, NzMessageService } from 'ng-zorro-antd/message';
+
+@Component({
+  selector: 'app-forgot-password',
+  standalone: true,
+  imports: [NzFormModule, NzInputModule, NzButtonModule, FormsModule, ReactiveFormsModule, NzGridModule, NzTypographyModule, NzSpaceModule, RouterModule, NzMessageModule],
```

Comment:
You can create a separate module for all these UI components so that you don't have to list all of them every time but just import one module.